documentation/en/default-lotus-config.toml

```diff
@@ -230,6 +230,17 @@
     # env var: LOTUS_CHAINSTORE_SPLITSTORE_HOTSTOREFULLGCFREQUENCY
     #HotStoreFullGCFrequency = 20
 
+    # HotStoreMaxSpaceTarget sets a target max disk size for the hotstore. Splitstore GC
+    # will run moving GC if disk utilization gets within a threshold (150 GB) of the target.
+    # Splitstore GC will NOT run moving GC if the total size of the move would get
+    # within 50 GB of the target, and instead will run a more aggressive online GC.
+    # If both HotStoreFullGCFrequency and HotStoreMaxSpaceTarget are set then splitstore
+    # GC will trigger moving GC if either configuration condition is met.
+    #
+    # type: uint64
+    # env var: LOTUS_CHAINSTORE_SPLITSTORE_HOTSTOREMAXSPACETARGET
+    #HotStoreMaxSpaceTarget = 0
```

Comment:
Can we give some minimum recommendation, e.g. in terms of a multiplier of the "lightweight" (currently 123GiB) snapshot size.

Something like 4-5x the snapshot size should work (would be good to test)

Reply:
From lots of recent operational experience we know that fully clean hotstore size is somewhere between 220 and 250 G.  So double this at 500GB is the absolute minimum, call it 550 GB with the safety buffer.  This turns out to be ~5x snapshot size so your intuition is good.

Reply:
The correspondence between snapshot size and hotstore size is somewhat confounded since snapshot = state + 3*(churn + messages) and hotstore = state + [between 4 and 5]*(churn + messages)